sdk/batch/microsoft-azure-batch/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 7.1.0-beta.1 (Unreleased)
```

Comment:
Why is `microsoft-azure-batch` changelog updated when form-recognizer is released? Also, `8.0.0` has been released, so `7.1.0-beta.1` seems incorrect.

@JimSuplizio

Reply:
@srnagar there's already a ticket filed against this and the bom's changelog below. Unfortunately, the update script picks scans all changelogs when inspecting versions and, if it finds one that is incorrect (like they are), it generates an entry.